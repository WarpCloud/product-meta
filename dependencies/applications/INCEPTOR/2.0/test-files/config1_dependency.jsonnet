{
  application_type: "INCEPTOR",
  application_name: "inceptor1",
  application_version: "2.0",
  user_config: {
    "use_hyperbase": "true",
    "use_search": "true",
  },
  app_config: {
  },
  instance_config: {
    "inceptor": {
      "version": "2.0",
      "Transwarp_Auto_Injected_Volumes": {

        },
    }
  },
  development_mode: true,
}
