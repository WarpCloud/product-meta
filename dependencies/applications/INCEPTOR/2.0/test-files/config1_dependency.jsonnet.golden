{
   "Customized_Namespace": "default",
   "Develop": true,
   "TCU": {
      "inceptor": {
         "c": 0.5,
         "m": 1,
         "s": 1
      },
      "metastore": {
         "c": 0.25,
         "m": 0.5,
         "s": 0.5
      }
   },
   "Transwarp_Install_ID": "",
   "instance_list": [
      {
         "annotations": {
            "application_name": "inceptor1",
            "application_type": "INCEPTOR",
            "use_hyperbase": "true",
            "use_search": "true"
         },
         "configs": {
            "metastore_cpu_limit": 0.5,
            "metastore_cpu_request": 0.5,
            "metastore_memory_limit": 1,
            "metastore_memory_request": 1,
            "mysql_cpu_limit": 0.5,
            "mysql_cpu_request": 0.5,
            "mysql_memory_limit": 1,
            "mysql_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            }
         ],
         "instance_settings": { },
         "moduleName": "metastore",
         "name": "inceptor1-metastore",
         "version": "1.0"
      },
      {
         "annotations": {
            "application_name": "inceptor1",
            "application_type": "INCEPTOR",
            "use_hyperbase": "true",
            "use_search": "true"
         },
         "configs": {
            "inceptor_executor_cpu_limit": 1,
            "inceptor_executor_cpu_request": 1,
            "inceptor_executor_memory_limit": 3,
            "inceptor_executor_memory_request": 3,
            "inceptor_master_cpu_limit": 1,
            "inceptor_master_cpu_request": 1,
            "inceptor_master_memory_limit": 1,
            "inceptor_master_memory_request": 1
         },
         "dependencies": [
            {
               "moduleName": "metastore",
               "name": "inceptor1-metastore"
            },
            {
               "moduleName": "hdfs",
               "name": "hdfs"
            },
            {
               "moduleName": "zookeeper",
               "name": "zookeeper"
            },
            {
               "moduleName": "hyperbase",
               "name": "hyperbase"
            },
            {
               "moduleName": "elasticsearch",
               "name": "elasticsearch"
            }
         ],
         "instance_settings": {
            "Transwarp_Auto_Injected_Volumes": { },
            "version": "2.0"
         },
         "moduleName": "inceptor",
         "name": "inceptor1-inceptor",
         "version": "2.0"
      }
   ],
   "instance_settings": { },
   "version": ""
}
